validation/src/test/scala/hmda/validation/rules/lar/validity/V215Spec.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V215Spec extends LarEditCheckSpec {
+  property("If action taken type must = 6, then date application received = NA") {
+    forAll(larGen) { lar =>
+      whenever(lar.loan.applicationDate == "NA") {
```

Comment:
Because you are only picking out one value from the generator it might make more sense to just use `lar.loan.copy(applicationDate = "NA")`.